src/view/dataElements/eventMergeId.jsx

```diff
@@ -32,29 +31,53 @@ const getSettings = ({ values }) => {
   return values;
 };
 
-const EventMergeId = () => {
+const EventMergeId = ({ initInfo, formikProps, registerImperativeFormApi }) => {
+  useEffect(() => {
+    registerImperativeFormApi({ getSettings });
+    formikProps.resetForm({ values: getInitialValues({ initInfo }) });
+  }, []);
+
+  // Formik state won't have values on the first render.
+  if (!formikProps.values) {
+    return null;
+  }
+
+  return (
+    <FillParentAndCenterChildren className="u-flexColumn">
+      <Alert variant="informative" title="Event Merge ID Caching">
+        This data element will provide an event merge ID. Regardless of what you
+        choose for the data element storage duration in Launch, the value of
+        this data element will remain the same until either the visitor to your
+        website leaves the current page or the event merge ID is reset using the
+        Reset Event Merge ID action.
+      </Alert>
+      <div className="EventMergeId-description u-gapTop2x">
```

Comment:
The `EventMergeId-description` styles were removed as part of deleting eventMergeId.styl, so the class name should be removed here or the styl file reinstated...or whatever you want to do as long as it looks decent.

Reply:
Thanks for catching this. I didn't realize we were changing the font size here.